microservices-connector/api/v1alpha3/gmconnector_types.go

```diff
@@ -15,6 +15,7 @@ import (
 type GMCTarget struct {
 	NameSpace string `json:"nameSpace,omitempty"`
 
+	// +optional
```

Comment:
shall this  "+optional"  be put before line 16 instead?

Reply:
yes, will resubmit after merging with #72